aviary/utils/preprocessors.py

```diff
@@ -303,16 +256,23 @@ def preprocess_propulsion(aviary_options: AviaryValues, engine_models: list = No
                 f'Mount location for engines of type <{eng_name}> not specified. '
                 'Wing-mounted engines are assumed.')
 
-        # If wing mount type are specified but inconsistent, default num_engines to sum
-        # of specified mounted engines
-        elif total_engines_calc != num_engines:
+        # If wing mount type are specified but inconsistent, handle it
+        elif total_engines_calc > num_engines:
+            # more defined engine locations than number of engines - increase num engines
             eng_name = engine.name
             num_engines_all[i] = total_engines_calc
             warnings.warn(
                 'Sum of aircraft:engine:num_fueslage_engines and '
                 'aircraft:engine:num_wing_engines do not match '
                 f'aircraft:engine:num_engines for EngineModel <{eng_name}>. Overwriting '
                 'with the sum of wing and fuselage mounted engines.')
+        elif total_engines_calc < num_engines:
+            # fewer defined locations than num_engines - assume rest are wing mounted
```

Comment:
if fewer engine locations are specified than there are engines, wouldn't it make more sense to assume that the unspecified ones are on the fuselage?

Reply:
My logic is that I think the most common scenario this will happen is when users set up their engines but forget to specify location. Since the majority of aircraft Aviary's methods are designed for have wing-mounted engines I chose that as the default. Since fuselage-mounted engines are typically the exception to the norm I'd expect users to be more likely to remember to specify mount location. Even more so for a mixed-mount engine setup.

Reply:
but if they haven't specified a location, what do we use as the value? it seems like the only time someone wouldn't bother specifying a wing location is if it isn't on the wing.
Maybe this should be an error then.

Reply:
Inherited from FLOPS, we have `NUM_ENGINES`, `NUM_WING_ENGINES`, and `NUM_FUSELAGE_ENGINES` - `NUM_ENGINES` is supposed to be the sum of wing and fuselage engines. I think it will be pretty common for users to incorrectly set up these values, either forgetting to specify one or more, or provide conflicting values. I set up catches for as many incorrect combinations as I could that are "salvageable", this one is for when the user only provided `NUM_ENGINES` and forgot to specify anything else. FLOPS defaults to wing-mounted in most cases, so I followed that logic.

Reply:
Do the span locations impact the wing weight calculation like they do with GASP? If we default to adding these as wing_engines, how do we decide where on the wing they go?